ddht/v5_1/alexandria/partials/proof.py

```diff
@@ -0,0 +1,144 @@
+from dataclasses import dataclass
+import operator
+from typing import Any, Collection, Iterable, Optional, Sequence, Tuple
+
+from eth_typing import Hash32
+from eth_utils import to_tuple
+from ssz.constants import CHUNK_SIZE, ZERO_HASHES
+from ssz.sedes import List as ListSedes
```

Comment:
Will finish reviewing this PR tmrw, but quick q: I might be blind, but I don't see any description of `sedes` in the ssz spec and there's very little documentation in the py-ssz lib. What are these things?

Reply:
It's a carry over from pyrlp.  I have no idea where the name "sedes" come from but you can think of it as a "schema" or "type" for how data is encoded.